README.md

```diff
@@ -22,3 +22,17 @@ $ helm install nginx-ingress-controller-app -f values.yaml
 ```
 
 Deployment to Tenant Clusters is handled by [app-operator](https://github.com/giantswarm/app-operator).
+
+## Dependencies
+
+### external-dns
+
+ExternalDNS synchronizes exposed Kubernetes Services and Ingresses with DNS providers. In a broader sense, ExternalDNS allows you to control DNS records dynamically via Kubernetes resources in a DNS provider-agnostic way.
+
+### kiam (applies to AWS provider only)
+
+'kiam' runs as an agent on each node in your Kubernetes cluster and allows cluster users to associate IAM roles to Pods.
+
+## Configuration
+
+Configuration options are documented in [Configuration.md](helm/nginx-ingress-controller-app/Configuration.md) document.
```

Comment:
I like this. I think having the docs in the app repo makes sense.

We should make this linkable from our UX. Is `home` set in the Chart.yaml? I like how @pipo02mix did this.

https://github.com/giantswarm/fluent-logshipping-app/blob/master/helm/fluent-logshipping-app-chart/Chart.yaml#L5

Reply:
Added `home`